### PR TITLE
[#467] Display useful snapshot metadata

### PR DESCRIPTION
**Pull request by krendelhoff2**
## Description

Problem: Since #441 tezos-setup-wizard is using xtz-shots.io metadata for node snapshot files. This metadata provides some information about snapshots e.g. snapshot head block hash, height, and timestamp.

Solution: Output some of it to user.

## Related issue(s)

Resolves #

#### Related changes (conditional)

- [ ] I checked whether I should update the [README](/serokell/tezos-packaging/tree/master/README.md)

- [ ] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: baking/src/tezos_baking/tezos_setup_wizard.py [hunk 1]
```diff
@@ -396,6 +396,19 @@ def get_snapshot_link(self):
...
             self.config["snapshot_block_hash"] = snapshot_metadata["block_hash"]
+            print(
+                color(
+                    f"""
+Snapshot metadata:
+url: {snapshot_metadata["url"]}
+sha256: {snapshot_metadata["sha256"]}
+filesize: {snapshot_metadata["filesize"]}
+block height: {snapshot_metadata["block_height"]}
+block timestamp: {snapshot_metadata["block_timestamp"]}
```
[the remark this answers is not in the document]
Comment (krendelhoff2) [reply baking/src/tezos_baking/tezos_setup_wizard.py:407]: what do you mean by localized?

[the remark this answers is not in the document]
Comment (pasqu4le) [reply baking/src/tezos_baking/tezos_setup_wizard.py:407]: I mean that it's printed in a way that uses the standard format of the running machine.

However, on second though, given that we only support English I think that it would be better to just use a format that is unambiguous, e.g. `Sep. 6th 2023`

[the remark this answers is not in the document]
Comment (pasqu4le) [reply baking/src/tezos_baking/tezos_setup_wizard.py:407]: @krendelhoff2 another slight change of plan after discussing with Oxhead.

Please disregard the above and let's use a date-time format that it's localized to the system running the wizard.

Other standard linux programs and utilities do the same, so this would make them easier to visually compare.

File: baking/src/tezos_baking/tezos_setup_wizard.py [hunk 2]
```diff
@@ -406,14 +403,47 @@ def hashes_comply(s1, s2):
...
             print(f"Couldn't collect snapshot metadata from {json_url}")
         except Exception as e:
             print(f"Unexpected error handling snapshot metadata:\n{e}\n")
 
+    def output_snapshot_metadata(self):
+        from datetime import datetime
+        from locale import setlocale, getlocale, LC_TIME
+
+        # it is portable `C` locale by default
+        setlocale(LC_TIME, getlocale())
```
Comment (pasqu4le) [inline baking/src/tezos_baking/tezos_setup_wizard.py:418]: Maybe I'm missing something, but why is this necessary?

Shouldn't the locale already be set on the machine? :thinking:

Comment (krendelhoff2) [reply baking/src/tezos_baking/tezos_setup_wizard.py:418]: It is set on machine, but python still trying to use portable locale by default
After that change, date became french on my `fr_FR.UTF-8` localized machine

Comment (pasqu4le) [reply baking/src/tezos_baking/tezos_setup_wizard.py:418]: Ok, I see that apparently this sets the global locale of the python app, which should be fine here.

**Review by pasqu4le — APPROVED** [review]
